catt/controllers.py

```diff
@@ -460,23 +463,26 @@ def kill(self, idle_only=False):
 
 
 class MediaControllerMixin:
+    _is_seekable: Any
```

Comment:
Basically just removing these two lines and the `_cast` below is good.

Reply:
Yeah, that SO answer has the right idea:


```
_is_seekable = None  # type: Any
```

By the way, isn't this a `bool`?

Reply:
> By the way, isn't this a bool?

Yes, but that doesn't work either. :sob:

Reply:
Hmm, it complains about the incompatibility, you mean?

Reply:
Yep

Reply:
Yeah, I hate mypy sometimes... Anyway, this looks good now, and I haven't been able to get it to pass by changing the bool type, so I think we're good to merge.

Reply:
Thanks!

Reply:
Thank you!